view-admin/src/components/common/JudgementModal/JudgementModal.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import styles from "./JudgementModal.module.css";
+import { RxCrossCircled } from "react-icons/rx";
+
+interface ModalProps {
+  isOpened: boolean;
+  isIncluded: boolean;
+  canCloseByClickingBackground?: boolean;
+  setisOpened: (isOpened: boolean) => void;
+  setIsIncluded: (included: boolean) => void;
+  setInputNumbers: (numbers: number[]) => void;
+  checkInclusion: () => void;
+}
+
+const Modal = ({
+  isOpened,
+  isIncluded,
+  canCloseByClickingBackground = true,
+  setisOpened,
+  setIsIncluded,
+  setInputNumbers,
+  checkInclusion,
+}: ModalProps) => {
+  const [inputValues, setInputValues] = useState<number[]>([0, 0, 0, 0, 0]);
+
+  const handleInputChange = (
+    index: number,
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    const newValue = parseInt(event.target.value);
+    const newInputValues = [...inputValues];
+    newInputValues[index] = newValue;
+    setInputValues(newInputValues);
+  };
+
+  const handleJugement = () => {
+    setTimeout(function(){
+      setInputNumbers(inputValues);
+      checkInclusion();
+    }, 500);
```

Comment:
```suggestion
  const handleJugement = () => {
    setInputNumbers(inputValues);
    checkInclusion();
```
index.tsxに解決策っぽいのつけたのでtimeout消した

Reply:
対応しました。